pages/linux/uuidgen.md

```diff
@@ -1,6 +1,8 @@
 # uuidgen
 
 > Generate unique identifiers (UUIDs).
+> See also `uuid`.
+> More information: <https://manned.org/uuidgen>.
 
 - Create a random UUID:
```

Comment:
Using `--random` we get an v4 UUID. Similarly below, using `--time` we get a v1 UUID and `--sha1 --namespace` is the v5 UUID (and `--md5 --namespace` would be v3 UUID).

Should there be version mentions here? Referencing to UUID by its version number seems to be very common around the internet.

Reply:
I could not add suggestions to the whole file as there in the middle are lines which were not modified with this commit (these lines need to be unfolded manually first and then they sadly break the blue vertical line).

Reply:
> Should there be version mentions here?

Sure, go ahead.

> I could not add suggestions to the whole file as there in the middle are lines which were not modified with this commit (these lines need to be unfolded manually first and then they sadly break the blue vertical line).

Why don't you use the GitHub editor or a local git repo to edit the files?

Reply:
Just wanted to learn the suggestion feature you just taught me, GitHub editor is still something I have not touched yet, one step at the time 👼🏻

Reply:
Yeah, sadly suggestions can only be made near other suggestions. (Kind of understanable. When working with code it doesn't make much sense suggesting differences to code which wasn't touched).

Reply:
I fully agree, I just did not know that. But then the only way to express my intentions before pushing a change is to do a comment, but here at this specific place a suggestion would work really well, even though outside of the active changes. Nevermind.

Reply:
While not as aesthetically pleasing, you could use

````
```diff
- before1
- before2
+ after1
+ after2
```
````
which will turn into 
```diff
- before1
- before2
+ after1
+ after2
```
along with the line number of the file you are referencing to.